package.json

```diff
@@ -39,8 +39,8 @@
     "moment": "^2.19.2",
     "request": "^2.88.0",
     "request-promise-native": "^1.0.7",
-    "@alertlogic/al-collector-js": "2.0.4",
-    "@alertlogic/al-aws-collector-js": "4.1.6"
+    "@alertlogic/al-collector-js": "3.0.4",
```

Comment:
could you please check that changing from 2.0.4 to 3.0.4 doesn't require any other code changes in this collector